bin/component-playground.js

```diff
@@ -15,4 +16,4 @@ var server = new WebpackDevServer(compiler, {
   hot: true
 });
 
-server.listen(8989, 'localhost', function() {});
+server.listen(userConfig.server.port, userConfig.server.hostname, function() {});
```

Comment:
This line has over 80 chars, and it failed the build: https://travis-ci.org/skidding/cosmos/builds/66354068#L401

(Will probably be fixed once you rename to `config`, though)